pulsar-broker/src/main/java/org/apache/pulsar/broker/BookKeeperClientFactoryImpl.java

```diff
@@ -6,9 +6,9 @@
  * to you under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p>
```

Comment:
I am not sure if this works. It might potentially fail the license header check.

Reply:
> I am not sure if this works. It might potentially fail the license header check.

I think it works, but it seems `mvn license:format` is not a right command to generate license for project, do you know what's the right one?